docs/policy-topology-aware.md

```diff
@@ -1,2 +1,4 @@
+# Topology aware policy
+
```

Comment:
It would have been nicer to distill this (and other similar fixes to the raw docs) into a separate 'docs: documentation fixes.'-commit. But it's fine like this now, no need to split it out any more.